ivre/nmapopt.py

```diff
@@ -123,6 +126,8 @@ def options(self):
         if self.verbosity:
             options.append('-%s' % ('v' * self.verbosity))
         options.extend(NMAP_OPT_PORTS.get(self.ports, ['-p', self.ports]))
+        if self.top_ports is not None:
+            options.extend(['--top-ports', self.top_ports])
```

Comment:
I **think** one might want to use
```Python
{
    top_ports: 5000
}
```
rather than
```Python
{
    top_ports: "5000"
}
```
So maybe this would be better:
```suggestion
            options.extend(['--top-ports', str(self.top_ports)])
```